binding-library/python/kafka/_kafka_abc.py

```diff
@@ -0,0 +1,28 @@
+import abc
+import typing
+
+class AbstractKafkaEvent(abc.ABC):
+
+    @abc.abstractmethod
+    def get_body(self) -> bytes:
+        pass
+    
+    @abc.abstractmethod
```

Comment:
nit: You should add `@property` decorator to this property and the ones below it. That would be a more appropriate for these along with the abstract method decorator.